client/src/view/components/Header/Nav/BurgerDropdown/Social/Social.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import { Link } from 'react-router-dom';
+// import Facebook from '../../../icons/facebook.svg';
```

Comment:
Does it work properly when images are commented?

Reply:
fixed